.github/utils/dryruns.py

```diff
@@ -21,6 +21,9 @@
     "stream aws-pcluster instance_type=c4.xlarge",
     "stream cscs-daint",
     "stream generic-x86",
+    "sparta-snl+openmp aws-pcluster instance_type=c6g.xlarge",
+    "sparta-snl+openmp aws-pcluster instance_type=c4.xlarge",
+    "sparta-snl+openmp generic-x86",
```

Comment:
FYI This is to fix dryrun error where this particular experiment is generated asking for too many cores for these particular systems.